app/views/menu_items/_form.html.slim

```diff
@@ -0,0 +1,13 @@
+= form_for menu_item, url: path, data: { "turbo-frame" => "_top" } do |f|
+  - if menu_item.errors.any?
+    ul
+      - menu_item.errors.full_messages.each do |msg|
+        li= msg
```

Comment:
I would suppose, same code You have almost on any form, so It could be extracted to application_helper